cmd/prompt_test.go

```diff
@@ -33,8 +33,8 @@ func Test_newPromptForCredentials(t *testing.T) {
 	go func() {
 		chars := expectedCreds.Username + enter + expectedCreds.Password + enter
 		for _, ch := range chars {
+			time.Sleep(time.Millisecond * 100)
```

Comment:
This test was failing at times, with a value for username as `estuser`.  My assmption was that it sent the first character, `t`, ever so slightly before the mock terminal was ready to accept.